### PR TITLE
docs: update set section

### PR DESCRIPTION
**Pull request by YangFong**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md [hunk 1]
```diff
@@ -19,17 +21,17 @@ mySet.clear();
...
 ```js
-var mySet = new Set();
+const mySet = new Set();
 mySet.add(1);
 mySet.add("foo");
 
-mySet.size;       // 2
-mySet.has("foo"); // true
+console.log(mySet.size);       // 2
+console.log(mySet.has("foo")); // true
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md:29]: This is not in the english document

File: files/zh-cn/web/javascript/reference/global_objects/set/delete/index.md [hunk 2]
```diff
@@ -6,31 +6,33 @@ slug: Web/JavaScript/Reference/Global_Objects/Set/delete
...
 
 ```js
-var mySet = new Set();
+const mySet = new Set();
 mySet.add("foo");
 
-mySet.delete("bar"); // 返回 false，不包含 "bar" 这个元素
-mySet.delete("foo"); // 返回 true，删除成功
+console.log(mySet.delete("bar")); // 返回 false，不包含 "bar" 这个元素
+console.log(mySet.delete("foo")); // 返回 true，删除成功
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/delete/index.md:33]: also

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by YangFong** [comment]
Ok, I'll try to change the English version.

**Comment by yin1999** [comment]
> Ok, I'll try to change the English version.

Please modify the english documents first then reflect the changes in translated-content

**Review by yin1999 — APPROVED** [review]

**Pull request merged**